### ISSUE TITLE
attention loss

### ISSUE THREAD

**Issue by AdeDZY** [QingyaoAi/Deep-Listwise-Context-Model-for-Ranking-Refinement#1]
Hi Qingyao, congratulations on the nice work! I have a question about the attention loss function:

`loss = tf.nn.softmax_cross_entropy_with_logits(logits=output, labels=target)
`
It looks different from the formula from the paper. Can you explain a little bit? Thanks!

**Comment by QingyaoAi** [comment]
Thanks! 

Sorry for the confusion. The input (i.e. “labels”) of this softmax function is actually an attention distribution computed with the exponentials of graded relevance labels. It is computed in line 227 in RankLSTM_model.py. I will refactor the code to make it clearer once I have time.

**Comment by AdeDZY** [comment]
AWESOME! THANKS!
